server/apps/main/templates/main/experiences_page.html

```diff
@@ -128,33 +128,32 @@ <h3 class="card-text">PERSON'S <br/> AUTISM </br> CONNECTION </h3>
 						<div class="row-sm-2 offset-sm-1 " style="width: 167px; height: 91px;background: #DD2424"></div>
 					</div>
 					<br><br><br>
+                    {% for i in experiences %}

```

Comment:
Don't use `i`. Use a more descriptive name than `i`, for example `experience`.

Reply:
Yes sure